worker/process-image.js

```diff
@@ -96,6 +105,31 @@ function uploadToS3 (s3, path, key, metadata, thumbPath, callback) {
   });
 }
 
+function translateJPEG2000 (ext, path, key, callback) {
+  if (ext === '.jp2') {
+    log(['debug'], 'Converting JPEG2000 to TIFF.');
+    var parsedPath = pathTools.parse(path);
+    var outPath = pathTools.join(parsedPath.dir, parsedPath.name) + '.tif';
+
+    var cmd = `${translateExe} -of GTiff ${path} ${outPath} ` +
+              '-co TILED=yes ' +
+              '-co COMPRESS=DEFLATE ' +
+              '-co PREDICTOR=2 ' +
+              '-co SPARSE_OK=yes ' +
+              '-co BLOCKXSIZE=256 ' +
+              '-co BLOCKYSIZE=256 ' +
+              '-co INTERLEAVE=band ' +
+              '-co NUM_THREADS=ALL_CPUS';
+    cp.execSync(cmd);
```

Comment:
In addition to errors thrown if the command can't be started, you should also pay attention to exit codes. Anything other than `0` means that the command failed. (Not sure if `exec` embeds that knowledge.)